suite-native/theme/src/useUserColorScheme.ts

```diff
@@ -0,0 +1,17 @@
+import { useAtom } from 'jotai';
+
+import { ThemeColorVariant } from '@trezor/theme';
+import { atomWithMMKV } from '@suite-native/storage';
+
+export type AppColorScheme = ThemeColorVariant | 'system';
+
+const userColorSchemeAtom = atomWithMMKV<AppColorScheme>('colorScheme', 'standard');
```

Comment:
I think you changed logic here little bit, it should set dark theme as default in case device is using dark mode. I think if you will set this `system` it should work as before.

Reply:
fixed ec7bdb40a